### PR TITLE
Add GCP Terraform scripts

### PR DESCRIPTION
**Pull request by ahmadnazeri**
Adding Terraform scripts to create GCP services and install Featureform and Featureform Quickstart.

### CODE REVIEW

File: terraform/gcp/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+3. Run ``terraform init``
+4. Run ``terraform plan``
+5. Run ``terraform apply -`` then type ``yes`` when prompted
+
+## Configure the Kubectl
+1. Run ``gcloud container clusters get-credentials $(terraform output -raw kubernetes_cluster_name) --region $(terraform output -raw region)`` in `gcp_services` folder
+
+## Install Featureform
+1. Run ``cd ../featureform``
+2. Run ``gcloud auth application-default login`` to give Terraform access to GCP
```
Comment (sdreyer) [inline terraform/gcp/README.md:23]: Does this need to be run a second time?

File: terraform/gcp/featureform/featureform.tf [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  set {
+    name  = "installCRDs"
+    value = "true"
+  }
+}
+
+resource "helm_release" "featureform" {
+  name = "featureform"
+  namespace = "default"
+  version = "v0.1.0"
```
Comment (sdreyer) [inline terraform/gcp/featureform/featureform.tf:24]: Do we need to set this? Can it just pull the latest version?

File: terraform/gcp/README.md (hunk 1 above)
Comment (ahmadnazeri) [reply terraform/gcp/README.md:23]: good catch :)

File: terraform/gcp/featureform/terraform.auto.tfvars [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Featureform
+## Cert Manager 
+cert_manager_version = "v1.8.0"
+
+## Featureform Core
+featureform_version     = "v0.1.0"
+featureform_hostname    = ""
```
Comment (sdreyer) [inline terraform/gcp/featureform/terraform.auto.tfvars:7]: Should we leave this blank so itll error when they try to run it if its not filled?

File: terraform/gcp/featureform/featureform.tf (hunk 2 above)
Comment (ahmadnazeri) [reply terraform/gcp/featureform/featureform.tf:24]: It isn't needed; I will remove it.

File: terraform/gcp/featureform/terraform.auto.tfvars (hunk 3 above)
Comment (ahmadnazeri) [reply terraform/gcp/featureform/terraform.auto.tfvars:7]: Should I remove it here and the default value as well?

Comment (sdreyer) [reply terraform/gcp/featureform/terraform.auto.tfvars:7]: Yeah, just force them to do it

**Review by sdreyer — APPROVED** [review]

**Pull request merged**